data/secscan_model/secscan_v4_model.py

```diff
@@ -25,8 +35,14 @@
 
 logger = logging.getLogger(__name__)
 
-# TODO(alecmerdler): Retrieve `/state` from security scanner API
-indexer_state = ""
+
+class IndexReportState(IntEnum):
+    """
+    Potential states of the claircore indexer for a given manifest.
+    """
+
+    INDEX_ERROR = -1
```

Comment:
Clair V4 returns these as numeric values?

Reply:
Ah, good catch! I was confused why the `securityworker` kept re-indexing manifests, this explains it!